lib/inferno/dsl/fhir_evaluator/rules/abstract_has_examples_rule.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
```

Comment:
From discussion - delete this file, note Steve's comment

Reply:
Deleted